### PR TITLE
Added Folders to README

### PR DESCRIPTION
**Pull request by Lucien950**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,41 @@ $ npm run generate
...
+
+- Assets
+  - CSS/SCSS Files
+  - Fonts
+    - One Folder per font
+      - all font files
+      - a font style sheet
+  - Images, the ones that are unique to each page
+  - SVG files
+  - Utils(not to be confused with ~\utils), idk either its a big mystery to me.
```
Comment (picklepower24) [inline README.md:60]: clarify? I wouldn't leave this in the final product lol

File: README.md [hunk 2]
```diff
@@ -47,6 +47,41 @@ $ npm run generate
...
+
+- Components
+  - Elements within pages, like notes, groups, questions, etc.
+- Layouts
+  - Boilerplates
+- Middleware
+  - a very special plugin that won't go into the plugins folder
```
Comment (picklepower24) [inline README.md:67]: be a bit more specific?

File: README.md [hunk 3]
```diff
@@ -47,6 +47,41 @@ $ npm run generate
...
+  - The popups basically
+  - All grouped by page
+- Static
+  - Sitewide images
+- Store
+  -Typescript files to locally store short term dynamic data, analogous to RAM
+- Tests 🤷🏻‍♂️
+- Types 
+  - All the typescript files
+- Utils (not to be confused with ~\assets\utils)
```
Comment (picklepower24) [inline README.md:83]: clarify

**Review by picklepower24 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DrCheddar) [reply README.md:60]: LOL o those are for like the vuesax import tbh they should just be included into ~/utils but havent gotten around to it

File: README.md (hunk 2 above)
Comment (DrCheddar) [reply README.md:67]: Middleware is for routing and handling authentication from the server side

File: README.md (hunk 3 above)
Comment (DrCheddar) [reply README.md:83]: Types is all the type declarations

**Review by DrCheddar — APPROVED** [review]
I like your sense of humour

**Review by picklepower24 — APPROVED** [review]

**Pull request merged**